utils.js

```diff
@@ -375,7 +375,8 @@ function formatDeltaMessage(m){
     messageID: md.messageId,
     attachments: (m.delta.attachments || []).map(v => _formatAttachment(v)),
     timestamp: md.timestamp,
-    isGroup: !!md.threadKey.threadFbId
+    isGroup: !!md.threadKey.threadFbId,
+    reactions: originalMessage.reactions ? originalMessage.reactions : []
```

Comment:
As far as I know, Delta message doesn't have info about reactions. But, the information about reaction is still in `/pull/` request, but marked as `ClientPayload`. So first you get `DeltaMessage` and then `ClientPayload` contains Message Reaction